data/en/cfabort.json

```diff
@@ -2,6 +2,7 @@
 	"name":"cfabort",
 	"type":"tag",
 	"syntax":"<cfabort>",
+	"script":"abort;",
```

Comment:
As far as I can see, your changes do not align with your pr's title: There's an additional semicolon,

Reply:
In cfdocs, cfabort tag script syntax displaying as `abort;;`(having double semicolons) in UI. by using this line I've avoided/ removed extra semicolon. please see the screenshot.
![double_semicons](https://user-images.githubusercontent.com/22448716/75804496-0cdd3780-5da6-11ea-9aba-9639ba5bfe29.PNG)